operator/src/main/java/oracle/kubernetes/operator/helpers/ConfigMapHelper.java

```diff
@@ -1,5 +1,7 @@
 // Copyright (c) 2018, 2022, Oracle and/or its affiliates.
 // Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+// Copyright (c) 2018, 2022, Oracle and/or its affiliates.
```

Comment:
This is odd... You've got two copyrights.

Reply:
Hmm, must be the hitting the wrong key